frontend/src/scenes/experiments/experimentLogic.test.ts

```diff
@@ -22,6 +25,68 @@ describe('experimentLogic', () => {
                 previous: null,
                 results: [{ id: 1, name: 'Test Exp', description: 'bla' }],
             }
+        } else if (
+            pathname === `api/projects/${MOCK_TEAM_ID}/experiments/${RUNNING_EXP_ID}` ||
+            pathname === `api/projects/${MOCK_TEAM_ID}/experiments/${RUNNING_FUNNEL_EXP_ID}`
+        ) {
+            return {
+                created_at: '2022-01-13T12:44:45.944423Z',
```

Comment:
maybe we can move these to a mock file?

Reply:
other response & results depend on the values here, so it felt easier for me to keep them in the same place 😅 (look above, see which value you care about, use the same value in the next result).

If these get out of hand though, makes sense to move them to their own files. But maybe not right now?